xamarin/Xamarin.Forms
# [UWP] Unfocus control before dispose (if have the focus)

### Description of Change ###

Unfocus control before dispose (if have the focus) on UWP.

### Issues Resolved ### 

- fixes #12716

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix12716](https://user-images.githubusercontent.com/6755973/102201691-bc634080-3ec6-11eb-86e9-e2836a036030.gif)

### Testing Procedure ###
Navigate to the issue 12716. Tap the Button to navigate to the second page. Set focus in the Editor.Press the back button in top left corner of the app. If a dialog appears, the test has passed.

### PR Checklist ###
